stock_picking_package_preparation/security/package_preparation_security.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="1">
+
+    <record model="ir.rule" id="pack_preparation_comp_rule">
+        <field name="name">Package Preparation multi-company</field>
+        <field name="model_id" ref="model_stock_picking_package_preparation"/>
+        <field name="global" eval="True"/>
```

Comment:
You shouldn't write to `global`, this is a function field which is true when the rule has no group.